jupyterlab-dataops/resources/cloudbeaver/set_metadata_run_jupyter.sh

```diff
@@ -0,0 +1 @@
+python /opt/cloudbeaver/set_metadata.py && /run_jupyter.sh
```

Comment:
This file is missing `run_jupyter.sh`? I cannot find this one

Reply:
That's inside `gcr.io/deeplearning-platform-release/base-cpu:m88` image.